sim_ether.c

```diff
@@ -470,8 +470,8 @@ t_stat eth_mac_scan_ex (ETH_MAC* mac, const char* strmac, UNIT *uptr)
       }
 
   /* final check - mac cannot be broadcast or multicast address */
-  if (!memcmp(newmac, zeros, sizeof(ETH_MAC)) ||  /* broadcast */
-      !memcmp(newmac, ones,  sizeof(ETH_MAC)) ||  /* broadcast */
+  if (!eth_mac_cmp(newmac, eth_mac_any) ||        /* broadcast */

```

Comment:
I agree that the all-zeroes address is not a reasonable value to set, but it isn't a broadcast address, obviously.